src/components/daydream-controls.js

```diff
@@ -83,7 +81,14 @@ module.exports.Component = registerComponent('daydream-controls', {
   },
 
   checkIfControllerPresent: function () {
-    var isPresent = this.isControllerPresent(this.el.sceneEl, GAMEPAD_ID_PREFIX, {hand: this.data.hand});
+    var handedness = this.data.hand === '' ? ['left', 'right'] : [this.data.hand];
```

Comment:
it doesn't seem right to me to go through isControllerPresent twice.  if you want to do your own processing, perhaps you should use getGamepadsByPrefix once instead?